src/GapicClientTrait.php

```diff
@@ -528,6 +528,92 @@ private static function defaultTransport()
             : 'rest';
     }
 
+    /**
+     * @param string $methodName
+     * @param string $interfaceName
+     * @param Message $request
+     * @param array $optionalArgs {
+     *     Call Options
+     *
+     *     @type array $headers [optional] key-value array containing headers
+     *     @type int $timeoutMillis [optional] the timeout in milliseconds for the call
+     *     @type array $transportOptions [optional] transport-specific call options
+     *     @type RetrySettings|array $retrySettings [optional] A retry settings
+     *           override for the call.
+     * }
+     *
+     * @return PromiseInterface|PagedListResponse|BidiStream|ClientStream|ServerStream
+     */
+    private function startApiCall(
+        $methodName,
+        $interfaceName = null,
```

Comment:
As of PHP 7.0 we can do this:
```suggestion
        string $methodName,
        string $interfaceName = null,
```